juriscraper/pacer/email.py

```diff
@@ -513,76 +513,83 @@ def _parse_bankruptcy_short_description(self, subject: str) -> str:
-            short_description = short_description.split(docket_number)[-1]
-
-            # Remove strings starting with "Ch" followed by a number
-            regex = r"\bCh\d+\b"
-            short_description = re.sub(regex, "", short_description)
-
-            # Remove docket number traces "-AAA"
-            regex = r"^-.*?\s"
-            short_description = re.sub(regex, "", short_description)
-        elif self.court_id in ["pawb", "ndb", "deb"]:
-            # In: Ch-7 22-20823-GLT U LOCK INC Reply
-            # Out: Reply
-            if case_name in subject:
-                # See deb_2.txt for need of this check
+        candidates = []
+        for docket_number, case_name in zip(
+            self.docket_numbers, self.case_names
+        ):
+            short_description = ""
+
+            if self.court_id in ["cacb", "ctb", "cob", "ianb"]:
+                # In: 6:22-bk-13643-SY Request for courtesy Notice of Electronic Filing (NEF)
+                # Out: Request for courtesy Notice of Electronic Filing (NEF)
+                short_description = subject.split(docket_number)[-1]
+
+                # Remove docket number traces "-AAA"
+                regex = r"^-.*?\s"
+                short_description = re.sub(regex, "", short_description)
+            elif self.court_id in ["njb", "dcb", "vaeb", "paeb", "mdb"]:
+                # In: Ch-11 19-27439-MBK Determination of Adjournment Request - Hollister Construc
+                # Out: Determination of Adjournment Request
+                short_description = subject.split(docket_number)[-1]
+                # Remove docket number traces "-AAA"
+                # Remove CH after docket and BK after short description for dcb
+                regex = r"^-.*?\s|C[Hh][\s\d]+|[ (]?B[Kk]( Other)?[) ]?"
+                short_description = re.sub(regex, "", short_description)
+                short_description = short_description.rsplit("-", 1)[0]
+            elif self.court_id == "nysb":
+                # In: 22-22507-cgm Ch13 Affidavit Re: Gerasimos Stefanitsis
+                # Out: Affidavit
+                if case_name not in subject:
+                    continue
+
+                short_description = subject.split(case_name)[0]
+                short_description = short_description.replace("Re:", "")
+                short_description = short_description.split(docket_number)[-1]
+
+                # Remove strings starting with "Ch" followed by a number
+                regex = r"\bCh\d+\b"
+                short_description = re.sub(regex, "", short_description)
+
+                # Remove docket number traces "-AAA"
+                regex = r"^-.*?\s"
+                short_description = re.sub(regex, "", short_description)
+            elif self.court_id in ["pawb", "ndb", "deb"]:
+                # In: Ch-7 22-20823-GLT U LOCK INC Reply
+                # Out: Reply
+                if case_name not in subject:
+                    continue
```

Comment:
Should we just do this at the top of the loop instead of in each branch? Why only in this and the other branch instead of all of them?

Reply:
The first 2 branches don't use the `case_name`, only the `docket_number`. So it makes sense to skip only on the branches that actually use the variable. If it is not skipped, the `subject.split(case_name)[0 or -1]` will return the whole subject